### PR TITLE
COG-1121-Accessibility issue on update password page

### PR DESCRIPTION
**Pull request by sunilb0575**
**User Story:**
- https://osi-cwds.atlassian.net/browse/COG-1121

**Description:**
- Fix for accessibility issues on update password and reset password pages.

### CODE REVIEW

File: src/components/PasswordInstructions.js [hunk 1]
```diff
@@ -14,17 +14,19 @@ const PasswordInstructions = props => {
     <div className='passwordIndicatorSection'>
       <span>Password must contain the following:</span><br/>
       <div className="col-xs-6">
-        <ul>
-          {props.validateLength ? Tick() : Cross()} Minimum 8 characters<br/>
-          {props.validateLowerCase ? Tick() : Cross()} Lower case letter [a-z]<br/>
-          {props.validateUpperCase ? Tick() : Cross()} Upper case letter [A-Z]<br/>
+        <ul className='PasswordInstructionsList'>
```
Comment (RamuKammagani-osi) [inline src/components/PasswordInstructions.js]: Where is this "PasswordInstructionsList" defined? i don't see it in cognito-login.css nor custom-css?

Comment (sunilb0575) [reply src/components/PasswordInstructions.js]: Forgot to remove, Done. Thanks.

**Comment by kelfink** [comment]
Why the force-pushes?  If you make a new change to the PR, just add it to the previous.  commits will be squashed at the end.  Force-pushing invalidates the comments made in the PR reviews.

**Comment by sunilb0575** [comment]
@kelfink To maintain one commit in the PR I rebase the branch only if there is no big change.  And as a result of rebasing I have to do force-push.

**Comment by sunilb0575** [comment]
Regarding invalidating comments, even in regular push case if we make any changes on the exact line on which there is a comment than that comment will be gone from the file changes, but comments will still be present in the conversation tab.

**Pull request merged**